### DISCUSSION TITLE
Scripts Kill SUDO

### DISCUSSION THREAD

**Discussion by wolframpaige** [usnistgov/macos_security#455] [Q&A] [answered]
After running these scripts sudo passwords are no longer except. What is causing this? Also I get lot of errors about empty spaces.

**Comment by jmahlman** [comment, marked as answer]
It would help to know what you created and ran. The scripts in the project don't perform any security changes, they create scripts and profiles that have to be deployed and run after testing. 

You most likely enabled enforcing smart cards. Have you tailored anything? Have you read the document in on what you created and ran?

**Comment by boberito** [comment]
Exactly what @jmahlman said. It sounds like you enforced the pam modules to only accept smart cards.
